float32 gain
---
bool feedback
